acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -130,10 +130,7 @@ public void completeCreationFlow(String apiKey, String org, String pipeline, boo
         }
         selectOrganization(org);
 
-        wait.until(singlePipelineBtn,120000)
```

Comment:
Well earlier it was single vs auto discovery and they needed organization to be selected. Now, there is already delay with selecting repo from drop down so I thought it was alright removing it. I will look at it tomorrow and if it helps will add delay.